### DISCUSSION TITLE
Is it possible to add children to a flow producer dynamically?

### DISCUSSION THREAD

**Discussion by devlzcode** [taskforcesh/bullmq#1766] [Q&A]
For example:
```ts
const flowProducer = new FlowProducer();

const flow = await flowProducer.add({
  name: 'parent',
  queueName: 'parents',
  children: [
    { name: 'child', queueName: 'children' },
  ],
});

// Inside the child worker processor
flow.addChild(child)
// or flow.addChildren([child, child, ...])
```
I know those aren't real APIs but is this on the roadmap? Also, is there anyway I can donate to bullmq or is that only through purchasing pro? Which is completely fine!

**Comment by roggervalf** [comment]
hi @agent-ly, thank you for your interested about it, currently we only can receive incomes by purchasing pro version for now. But returning about your question, I think you can check this pattern https://docs.bullmq.io/patterns/process-step-jobs#chaining-flows, hope it could help you
